Tabloid/client/src/components/ApplicationViews.js

```diff
@@ -26,7 +27,7 @@ export default function ApplicationViews({ isLoggedIn }) {
           <CategoryForm />
         </Route>
         <Route path="/categories/delete/:id" exact>
```

Comment:
Does this route need to be exact?

Reply:
Should the id param in this route be an integer? if so, please make sure it's an integer by adding regex to the param. See my suggestion ...

Reply:
```suggestion
        <Route path="/categories/delete/:id(\d+)">
```

Reply:
👍🏻